app/controllers/api/profile_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Api::ProfileController < ApiController
+  skip_before_action :validate_user
+  before_action :skip_authorization
+
+  self.permitted_attrs = [:preferred_locale]
+
+  def update
```

Comment:
maybe there is an elegant way to accomplish this with dry crud? 
check docs at https://github.com/codez/dry_crud
sync with Lian ... he already red some things about dry crud. 

hint: probably override the def entry method for customizing this.